offscreen-canvas/the-offscreen-canvas/offscreencanvas.commit.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/canvas-tests.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#the-offscreen-2d-rendering-context">
```

Comment:
Also applicable to other files in this PR: avoid referencing the multipage version of the spec since those links are more likely to break if the spec sections get re-organized.